internal/configs/virtualserver_test.go

```diff
@@ -8102,7 +8216,7 @@ func TestIsTLSEnabled(t *testing.T) {
 	}
```

Comment:
I added a test case to cover nsmEgress being set to true. You are right in that it should return false. nsmEgress and TLS Enable are conflicting configuration items so nsmEgress will need to take precedence.

Reply:
Having a user set both shouldn't really matter. They are still either enabling the internal route for NSM or not. The underlying configuration would be invisible to the user. That being said, I can add a validation check if you believe it would be valuable to users.